datafusion/src/dataframe.rs

```diff
@@ -223,6 +223,21 @@ pub trait DataFrame: Send + Sync {
     /// ```
     async fn collect(&self) -> Result<Vec<RecordBatch>>;
 
+    /// Print results.
+    ///
+    /// ```
```

Comment:
We could add `no_run` here and remove the code comment there, so the doc string can still be tested to make sure it always compiles.

Reply:
I copied the code comment of the previous function, do I need to add it here?

Reply:
ha, ok, never mind then :)